doc/topics/cloud/vmware.rst

```diff
@@ -0,0 +1,163 @@
+===================================
+Getting Started With VMware vSphere
+===================================
+
+VMware vSphere is a server virtualization platform for building cloud
+infrastructure.
+
+
+Dependencies
+============
+The vmware module for Salt Cloud requires the ``pyVmomi`` package, which is
+available at PyPI:
+
+https://pypi.python.org/pypi/pyvmomi
+
+This package can be installed using `pip` or `easy_install`:
+
+.. code-block:: bash
+
+    pip install pyvmomi
+    easy_install pyvmomi
+
+
+Configuration
+=============
+The VMware cloud module needs the vCenter URL, username and password to be
+set up in the cloud configuration at
+``/etc/salt/cloud.providers`` or ``/etc/salt/cloud.providers.d/vmware.conf``:
+
+.. code-block:: yaml
+
+    my-vmware-config:
+      provider: vmware
+      user: DOMAIN\user
+      password: verybadpass
+      url: vcenter01.domain.com
+
+    vmware-vcenter02:
+      provider: vmware
+      user: DOMAIN\user
+      password: verybadpass
+      url: vcenter02.domain.com
+
+    vmware-vcenter03:
+      provider: vmware
+      user: DOMAIN\user
+      password: verybadpass
+      url: vcenter03.domain.com
+
+
+Profiles
+========
+Set up an initial profile at ``/etc/salt/cloud.profiles`` or
+``/etc/salt/cloud.profiles.d/vmware.conf``:
+
+.. code-block:: yaml
+
+    vmware-centos6.5:
+      provider: vmware-vcenter01
+      clonefrom: test-vm
```

Comment:
@nmadhok is currently working on the functionality that will provide the `image` option. `clonefrom` performs a different function from it.